protocol/test/forge/v2/templeLineOfCredit/TempleLineOfCreditV2.t.sol

```diff
@@ -109,19 +90,24 @@ contract TempleLineOfCreditTest_Admin is TlcBaseTest {
         assertEq(address(tlc.tlcStrategy()), address(tlcStrategy));
         assertEq(address(tlc.treasuryReservesVault()), address(trv));
 
-        TreasuryReservesVault newTrv = new TreasuryReservesVault(rescuer, executor, address(templeToken), address(daiToken), address(dUSD), address(tpiOracle));
+        TreasuryReservesVault newTrv = new TreasuryReservesVault(rescuer, executor, address(tpiOracle));
```

Comment:
function `test_invalidDebtToken()` name is misleading , this is the same interest rate model set in `TlcBaseTest.t.sol` setup.

Reply:
Fixed in https://github.com/TempleDAO/temple/pull/814/commits/83e73237dda37b171b542607df5d4f4da06b5139